readme.md

```diff
@@ -106,6 +106,21 @@ Default: `process.cwd()`
 
 Directory to start from.
 
+##### type
```

Comment:
I'm not sure how you want to document that these options do not apply when a matcher function is used?

Reply:
Just add a note about it on line 97.